cloudvolume/volumecutout.py

```diff
@@ -0,0 +1,92 @@
+import os
```

Comment:
what does this special class for?

Reply:
It's what CloudVolume returns from downloads. It's derived from np.ndarray and you can use it the same way. It comes with a few helper items that track where it came from and a way to serialized images to disk for inspection.